Document/0x04i-Testing-User-Privacy-Protection.md

```diff
@@ -2,72 +2,72 @@
 
-Most parts of the processes derived from the protection goals are traditionally covered in a privacy policy. However, this approach is not always optimal:
+Though a privacy policy traditionally protects most of the these processes, that approach is not always optimal because:
 
-- developers are not legal experts but still need to be compliant.
-- users would be required to read usually long and wordy policies.
+- Developers are not legal experts but still need to be compliant with legislation.
+- Users almost always have to read long and wordy policies.
 
-### The New Approach (Google's and Apple's take on this)
+### The New Approach (Google and Apple)
 
-In order to address these challenges and help users easily understand how their data is being collected, handled, and shared, Google and Apple introduced new privacy labeling systems (very much along the lines of NIST's proposal for [Consumer Software Cybersecurity Labeling](https://nvlpubs.nist.gov/nistpubs/CSWP/NIST.CSWP.02042022-1.pdf):
+In order to address these challenge and better inform users, Google and Apple have introduced new privacy labeling systems (very much along the lines of NIST's proposal) to help users easily understand how their data is being collected, handled, and shared, [Consumer Software Cybersecurity Labeling](https://nvlpubs.nist.gov/nistpubs/CSWP/NIST.CSWP.02042022-1.pdf). Their approaches can be seen at: 
 
-- the App Store [Nutrition Labels](https://www.apple.com/privacy/labels/) (since 2020).
-- the Google Play [Data Safety Section](https://developer.android.com/guide/topics/data/collect-share) (since 2021).
+- The App Store [Nutrition Labels](https://www.apple.com/privacy/labels/) (since 2020).
+- The Google Play [Data Safety Section](https://developer.android.com/guide/topics/data/collect-share) (since 2021).
 
-As a new requirement on both platforms, it's vital that these labels are accurate in order to provide user assurance and mitigate abuse.
+Since this is a new requirement on both platforms, these labels must be accurate in order to reassure users and mitigate abuse.
 
-### Google ADA MASA program
+### Google ADA MASA Program
 
-Performing regular security testing can help developers identify key vulnerabilities in their apps. Google Play will allow developers who have completed independent security validation to showcase this on their Data safety section. This helps users feel more confident about an app's commitment to security and privacy.
+Since regular security testing helps developers identify key vulnerabilities in their apps, Google Play will allow developers who have completed independent security validation to inform users by disclosing this fact in the app's Data Safety section. The developer's commitment to security and privacy is meant to reassure users. 
```

Comment:
Markdown lint fix.
```suggestion
Since regular security testing helps developers identify key vulnerabilities in their apps, Google Play will allow developers who have completed independent security validation to inform users by disclosing this fact in the app's Data Safety section. The developer's commitment to security and privacy is meant to reassure users.
```